### PR TITLE
virtio_fs: add xattr and no_killpriv_v2 to xfstest

### PR DESCRIPTION
**Pull request by xiagao**
Add options for xfstest.
ID: 2150189
Signed-off-by: Xiaoling Gao <xiagao@redhat.com>

### CODE REVIEW

**Comment by xiagao** [comment]
@zhenyzha @YongxueHong Call for your review,thanks.

File: qemu/tests/cfg/virtio_fs_share_data_multi_backend.cfg [hunk 1]
```diff
@@ -189,6 +187,11 @@
                     image_snapshot = yes
                     mem = 32768
                     size_mem1 = 32G
+                    # Clearing the private setting of falocate and support xattr
+                    Host_RHEL.m9:
+                        fs_binary_extra_options += " --no-killpriv-v2 --xattr --modcaps=+sys_admin"
+                    Host_RHEL.m8:
+                        fs_binary_extra_options += " -o no-killpriv-v2 -o xattr -o modcaps=+sys_admin"
```
Comment (YongxueHong) [inline qemu/tests/cfg/virtio_fs_share_data_multi_backend.cfg:194]: Hi @xiagao 
From the codes, I think there are different ways to handle the options between RHEL9 and RHEL8, right?
or it is a kind of special situation？

Comment (xiagao) [reply qemu/tests/cfg/virtio_fs_share_data_multi_backend.cfg:194]: Thanks reminder. I think there is no need this time, I will send another pr to modify all together. Right now it's compatible with two options on rhel9.

Comment (YongxueHong) [reply qemu/tests/cfg/virtio_fs_share_data_multi_backend.cfg:194]: Okay, it looks good to me.

**Review by YongxueHong — DISMISSED** [review]
LGTM.

**Comment by zhenyzha** [comment]
`[stdlog] 2022-12-18 21:38:46,748 avocado.virttest.qemu_devices.qdevices INFO | Running virtiofs daemon command /usr/libexec/virtiofsd --socket-path=/var/tmp/avocado_2363mse0/avocado-vt-vm1-fs1-virtiofsd.sock -o source=/tmp/virtio_fs1_test -o cache=auto --no-killpriv-v2 --xattr --modcaps=+sys_admin`

The code executes as expected.

LGTM.Acked-by: Zhenyu Zhang <zhenyzha@redhat.com>

**Review by zhenyzha — APPROVED** [review]

**Review by YongxueHong — APPROVED** [review]

**Pull request merged**